src/main/java/com/blockout/controls/TextField.java

```diff
@@ -380,17 +381,17 @@ else if (cursorBeforeEnd)
             GlStateManager.disableTexture2D();
             GlStateManager.enableColorLogic();
             GlStateManager.colorLogicOp(GL11.GL_OR_REVERSE);
-            WorldRenderer worldrenderer = tessellator.getWorldRenderer();
+            VertexBuffer VertexBuffer = tessellator.getBuffer();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)